.golangci.yaml

```diff
@@ -12,6 +12,7 @@ linters:
   disable:
   - maligned
   - scopelint
+  - gci
```

Comment:
why this change? imo definitely helps with formatting

Reply:
@timraymond and I were discussing removing this as it messes up imports ordering in a way that's not necessarily "better" in our opinion. But open to discussion.